openstf/tasks/create_components_forecast.py

```diff
@@ -29,21 +29,78 @@
 
 
 """
-from openstf.pipeline.create_forecast import make_components_prediction
+import structlog
+from datetime import datetime, timedelta
+
+from openstf.pipeline.create_component_forecast import (
+    create_components_forecast_pipeline,
+)
 from openstf.tasks.utils.utils import check_status_change, update_status_change
 from openstf.tasks.utils.predictionjobloop import PredictionJobLoop
 from openstf.tasks.utils.taskcontext import TaskContext
 
+T_BEHIND_DAYS = 0
+T_AHEAD_DAYS = 3
+
 
-def create_components_forecast_pj(pj, context):
+def create_components_forecast_task(pj, context):
+    """Top level task that creates a components forecast.
+    On this task level all database and context manager dependencies are resolved.
+
+    Args:
+        pj (dict): Prediction job
+        context (TaskContext): Contect object that holds a config manager and a database connection
+    """
+    logger = structlog.get_logger(__name__)
     if pj["train_components"] == 0:
```

Comment:
I guess this name is a little misleading as well. `train_components` also means don't create a components forecast. But I guess it makes somewhat sense.

Reply:
On the other hand we do not even train components or models for components right? Now I m confused :)